server2/Arcadia.Assistant/Arcadia.Assistant.AppCenterBuilds/DownloadApplicationSettings.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Arcadia.Assistant.AppCenterBuilds.Contracts;
+using System.Fabric.Description;
+
+namespace Arcadia.Assistant.AppCenterBuilds
+{
+    public class DownloadApplicationSettings : IDownloadApplicationSettings
+    {
+        public DownloadApplicationSettings(ConfigurationSection configurationSection)
+        {
+            this.ApiToken = configurationSection.Parameters["ApiToken"].Value;
+            this.DownloadBuildIntervalMinutes = int.TryParse(configurationSection.Parameters["DownloadBuildIntervalMinutes"].Value, out var res) ? (int?)res : null;
+            this.AndroidGetBuildsUrl = configurationSection.Parameters["AndroidGetBuildsUrl"].Value;
+            this.AndroidGetBuildDownloadLinkTemplateUrl = configurationSection.Parameters["AndroidGetBuildDownloadLinkTemplateUrl"].Value;
+            this.IosGetBuildsUrl = configurationSection.Parameters["IosGetBuildsUrl"].Value;
+            this.IosGetBuildDownloadLinkTemplateUrl = configurationSection.Parameters["IosGetBuildDownloadLinkTemplateUrl"].Value;
+        }
+
+        public int? DownloadBuildIntervalMinutes { get; set; }
```

Comment:
should not be nullables. Just throw an error if something is invalid

Reply:
Done